code/datums/flock/flock.dm

```diff
@@ -453,6 +453,12 @@
 		count++
 	return count
 
+/datum/flock/proc/toggleDeconstructionFlag(var/atom/target)
+	if(target in src.deconstruct_targets)
+		src.deconstruct_targets -= target
+	else
+		src.deconstruct_targets += target
+	src.updateAnnotations()
```

Comment:
I'm certain pali or waka will complain that this isn't a macro

Reply:
is there a macro for this?

Reply:
No but proc calls are slow or something :shelterfrog:

Reply:
togglePriorityTurfs() does this, plus see annotations changes I made for togglePriorityTurfs(), would be making similar changes once this is merged